src/plugins/intel_cpu/src/transformations/cpu_opset/arm/pass/convert_conv_bias.cpp

```diff
@@ -0,0 +1,84 @@
+#include "transformations/utils/utils.hpp"
+#include "utils/cpu_utils.hpp"
+#include "utils/general_utils.h"
+
+using namespace ov::pass;
+
+ov::intel_cpu::ConvertConvolutionBias::ConvertConvolutionBias() {
+    auto conv_i8_activation = pattern::any_input(pattern::type_matches(element::i8));
+    auto conv_i8_weights = pattern::any_input(pattern::type_matches(element::i8));
+    auto conv_i8 = pattern::wrap_type<ov::op::v1::Convolution>({conv_i8_activation, conv_i8_weights});
+
+    auto conv_u8_activation = pattern::any_input(pattern::type_matches(element::u8));
+    auto conv_i8_u8_weights = pattern::any_input(pattern::type_matches_any({element::i8, element::u8}));
+    auto conv_u8 = pattern::wrap_type<ov::op::v1::Convolution>({conv_u8_activation, conv_i8_u8_weights});
+    auto conv_m = conv_u8 | conv_i8;
+
+    auto bias_const_m = pattern::wrap_type<ov::op::v0::Constant>();
+    auto add_m = pattern::wrap_type<ov::op::v1::Add>({conv_m, bias_const_m});
+    auto multiply_m = pattern::wrap_type<ov::op::v1::Multiply>({add_m, pattern::any_input()});
+
+    ov::matcher_pass_callback callback = [=](pattern::Matcher& m) {
+        auto mul = ov::as_type_ptr<ov::op::v1::Multiply>(m.get_match_root());
+        if (!mul) {
+            return false;
+        }
+        // mark Multiply as dequantization node to avoid its conversion to PowerStatic
+        ov::mark_as_dequantization_node(mul);
+
+        const auto& pattern_map = m.get_pattern_value_map();
+        auto bias_const = ov::as_type_ptr<ov::op::v0::Constant>(pattern_map.at(bias_const_m).get_node_shared_ptr());
+        // the transformation is not applied if add constant is already i32
+        if (bias_const->get_output_element_type(0) == ov::element::i32) {
+            return false;
+        }
+
+        // round the constant before converting to improve accuracy
+        auto round = std::make_shared<ov::op::v5::Round>(bias_const, ov::op::v5::Round::RoundMode::HALF_TO_EVEN);
+        auto convert_to_i32 = std::make_shared<ov::op::v0::Convert>(round, ov::element::i32);
+
+        auto add = pattern_map.at(add_m).get_node_shared_ptr();
+        // Check if the Add node is TypeRelaxed, if not - create a TypeRelaxed Add node
+        if (!std::dynamic_pointer_cast<ov::op::TypeRelaxedBase>(add)) {
+            auto conv_output = pattern_map.at(conv_m);
+            auto new_add = std::make_shared<ov::op::TypeRelaxed<ov::op::v1::Add>>(
+                ov::element::TypeVector{ov::element::f32, ov::element::f32},
+                ov::element::TypeVector{ov::element::f32},
+                ov::op::TemporaryReplaceOutputType(conv_output, ov::element::f32).get(),
+                ov::op::TemporaryReplaceOutputType(convert_to_i32->output(0), ov::element::f32).get());
+            ov::copy_runtime_info({add, bias_const}, {round, convert_to_i32, new_add});
+            ov::replace_node(add, new_add);
+        } else {
+            add->input(1).replace_source_output(convert_to_i32->output(0));
```

Comment:
What if we will always create type relaxed Add node? In this case, we avoid check on TypeRelaxed and additional code, and avoid issues in case of `ov::replace_output_update_name` usage. What do you think?

Reply:
I replace the current implementation with `ov::copy_runtime_info` + `ov::replace_node`. Does it work?

Reply:
Yes, this part looks good now, thank you